app/assets/javascripts/set-ga-client-id-on-url-in-element.js

```diff
@@ -0,0 +1,23 @@
+(function () {
+  'use strict'
+
+  window.GOVUK = window.GOVUK || {}
+  var GOVUK = window.GOVUK
+
+  function SetGaClientIdOnUrlInElement(options) {
+    if (!options.$linkedElement || !options.attribute || !window.ga) {
+      return
+    }
+
+    var linkedElement = options.$linkedElement
+    var attribute = options.attribute
+
+    window.ga(function (tracker) {
+      var clientId = tracker.get('clientId')
+      var href = linkedElement.attr(attribute)
```

Comment:
Is it confusing to call this `href` when it depends on the attribute itself (so might not always be a href)?

Reply:
Good point - [fixed](https://github.com/alphagov/government-frontend/pull/1744/commits/2286ccabb85d83bfaaf5e5cefabc4f2873f7832e)